docs/tr1/INSTALLING.md

```diff
@@ -437,6 +437,7 @@ If you install everything correctly, your game directory should look more or les
         │   │   ├── cistern_fd.bin
         │   │   ├── cistern_itemrots.bin
         │   │   ├── cistern_plants.bin
+        │   │   ├── cistern_skybox.bin
```

Comment:
BTW I noticed there's a couple of missing files here – just FYI there's a script `tools/update_install_trees` that updates these lists automatically for both TR1 and TR2.